contrib/cirrus/test.sh

```diff
@@ -67,7 +67,13 @@ else
             showrun make validate
             ;;
         unit)
-            showrun make test-unit
+	    race=
```

Comment:
nit, line indentation still seems wrong, looks like you use one tab and spaces on that line.

Reply:
fixed.